repos/system_upgrade/el7toel8/actors/cupsfiltersmigrate/libraries/library.py

```diff
@@ -0,0 +1,115 @@
+from leapp.libraries.common.rpms import has_package
+from leapp.libraries.stdlib import api
+from leapp.models import InstalledRedHatSignedRPM
+
+# rpm : the default config file
+BROWSED_CONFIG = '/etc/cups/cups-browsed.conf'
+
+
+# list of macros that should be set
+NEW_MACROS = [
+    'LocalQueueNamingRemoteCUPS RemoteName',
+    'CreateIPPPrinterQueues All'
```

Comment:
As discussed, it should not be issue as those macros are not on RHEL 7 system. However it will be better to check really just macros in such case and not string 'Macro Value'. So the list could be defined e.g. like that:
```diff
- NEW_MACROS = [
-    'LocalQueueNamingRemoteCUPS RemoteName',
-    'CreateIPPPrinterQueues All'
+ NEW_MACROS = [
+    ('LocalQueueNamingRemoteCUPS', 'RemoteName'),
+    ('CreateIPPPrinterQueues', 'All')
```

@zdohnal  Let's sync on monday.

Reply:
Just to be precise - the strings need to be checked in the case when configuration file is not modified by user before, so dnf transaction went well during upgrade and there is no .rpmnew file.